services/tenant-ui/frontend/src/assets/_theme.scss

```diff
@@ -5694,3 +6465,24 @@ $tenant-ui-shadow: lighten( $tenant-ui-secondary-color, 30% );
 }
 
 /* Customizations to the designer theme should be defined here */
+
+/* I had to put these here for expedience, i needed to delete a componant that accidentially added these to global scope */
+/* >>>>>>>>>>> */
+.row {
+  display: flex;
+  flex-direction: row;
+  margin: 10px;
+}
+.p-dialog-content input {
+  min-width: 400px;
+}
+.field > label,
+.field > small.p-error {
+  display: block !important;
+}
+
+.field > label > .p-progress-spinner {
+  height: 1em;
+  width: 1em;
+}
+/* <<<<<<<<<<< */
```

Comment:
I'd keep these out of the theme file (there's the style.css and we can do some reconfigure of that to be scss as well)